### PR TITLE
Fix mailto transport testcase failures

### PR DESCRIPTION
**Pull request by chanikag**
## Purpose
This PR is to fix mailto transport testcase failures

### CODE REVIEW

File: integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/mailto/transport/receiver/test/MailToTransportFolderTestCase.java [hunk 1]
```diff
@@ -36,17 +39,20 @@
...
     @BeforeClass(alwaysRun = true)
     public void initialize() throws Exception {
         super.init();
-        loadESBConfigurationFromClasspath(
-                File.separator + "artifacts" + File.separator + "ESB" + File.separator + "mailTransport"
-                        + File.separator + "mailTransportReceiver" + File.separator + "mail_transport_folder.xml");
-        logViewerClient = new LogViewerClient(contextUrls.getBackEndUrl(), getSessionCookie());
+        OMElement mailToProxyOMElement = AXIOMUtil.stringToOM(FileUtils.readFileToString(new File(
+                getESBResourceLocation() + File.separator + "mailTransport" + File.separator +
+                        "mailTransportReceiver" + File.separator + "mail_transport_folder.xml")));
```
Comment (a5anka) [inline integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/mailto/transport/receiver/test/MailToTransportFolderTestCase.java]: `+` should precede in the new line according to the followed code style. Maybe the IDE config is not correct.

File: integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/mailto/transport/receiver/test/MailToTransportInvalidAddressTestCase.java [hunk 2]
```diff
@@ -35,16 +38,18 @@
...
         super.init();
-        loadESBConfigurationFromClasspath(
-                File.separator + "artifacts" + File.separator + "ESB" + File.separator + "mailTransport"
-                        + File.separator + "mailTransportReceiver" + File.separator
-                        + "mail_transport_invalid_address.xml");
-        logViewerClient = new LogViewerClient(contextUrls.getBackEndUrl(), getSessionCookie());
+        OMElement mailToProxyOMElement = AXIOMUtil.stringToOM(FileUtils.readFileToString(new File(
+                getESBResourceLocation() + File.separator + "mailTransport" + File.separator +
+                        "mailTransportReceiver" + File.separator + "mail_transport_invalid_address.xml")));
+        Utils.deploySynapseConfiguration(mailToProxyOMElement,
```
Comment (a5anka) [inline integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/mailto/transport/receiver/test/MailToTransportInvalidAddressTestCase.java]: + should precede in the new line according to the followed code style. Maybe the IDE config is not correct.

File: integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/mailto/transport/receiver/test/MailToTransportInvalidFolderTestCase.java [hunk 3]
```diff
@@ -32,16 +35,18 @@
...
     public void initialize() throws Exception {
         super.init();
-        loadESBConfigurationFromClasspath(
-                File.separator + "artifacts" + File.separator + "ESB" + File.separator + "mailTransport"
-                        + File.separator + "mailTransportReceiver" + File.separator
-                        + "mail_transport_invalid_folder.xml");
-        logViewerClient = new LogViewerClient(contextUrls.getBackEndUrl(), getSessionCookie());
+        OMElement mailToProxyOMElement = AXIOMUtil.stringToOM(FileUtils.readFileToString(new File(
+                getESBResourceLocation() + File.separator + "mailTransport" + File.separator +
+                        "mailTransportReceiver" + File.separator + "mail_transport_invalid_folder.xml")));
```
Comment (a5anka) [inline integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/mailto/transport/receiver/test/MailToTransportInvalidFolderTestCase.java]: + should precede in the new line according to the followed code style. Maybe the IDE config is not correct.

Comment (chanikag) [reply integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/mailto/transport/receiver/test/MailToTransportInvalidFolderTestCase.java]: Done in https://github.com/wso2/micro-integrator/pull/575

File: integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/mailto/transport/receiver/test/MailToTransportInvalidAddressTestCase.java (hunk 2 above)
Comment (chanikag) [reply integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/mailto/transport/receiver/test/MailToTransportInvalidAddressTestCase.java]: Done in #575

File: integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/mailto/transport/receiver/test/MailToTransportFolderTestCase.java (hunk 1 above)
Comment (chanikag) [reply integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/mailto/transport/receiver/test/MailToTransportFolderTestCase.java]: Done in #575

**Pull request merged**